### ISSUE TITLE
gtk+ failed to build on linuxbrew standalone Ubuntu 14.04

### ISSUE THREAD

**Issue by arunnair411** [Linuxbrew/linuxbrew#632]
As part of getting opencv3 up and running, I was trying to install gtk+ through linuxbrew. It includes cairo as a dependency, which it installs displaying

```
==> ./configure --prefix=/home/arun/.linuxbrew/Cellar/cairo/1.14.2_1 --enable-gobject=yes --enable-svg=yes --enable-tee=yes --enable-xcb=no --enable-xlib=no --enable-xlib-xrender=no
```

Looking at cairo.rb [at](https://github.com/Homebrew/linuxbrew/blob/master/Library/Formula/cairo.rb), I see that enable-xlib is set to no as the x11 dependency is not satisfied. However, later in the installation process, 

```
configure: error: Package requirements (cairo-xlib >= 1.6) were not met:

No package 'cairo-xlib' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables CAIRO_BACKEND_CFLAGS
and CAIRO_BACKEND_LIBS to avoid the need to call
```

which seems to imply that enable-xlib should be true. How do I fix this? Here is the [gist log](https://gist.github.com/anonymous/57ef49a8346d46408fec)

**Comment by arunnair411** [comment]
Any thoughts as to what I can do to get it working?

**Comment by ahundt** [comment]
I don't think the linuxbrew x11 is what should be used here either, perhaps it should be the one provided by the system, even in the standalone case? Can x11 even be compiled and run properly in a truly standalone way? I feel like at some level dependencies must exist to the underlying system.

**Comment by DoomHammer** [comment]
Running `brew install cairo --with-x11` solves the issue. I do have, however `linuxbrew/xorg/xorg` installed locally.

**Comment by sjackman** [comment]
See also #542.
Fixed by #974.
